# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
#
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
#
# ROCK Linux: rock-src/package/base/linux24/80-ext23fs-suiddir.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
#
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
#
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
#
# --- ROCK-COPYRIGHT-NOTE-END ---

Userspace applications including kernel headers might utilize gcc
options like ANSI resulting in parse errors when other headers (e.g.
byteorder.h) utilize the __u64 type ...

 - Rene Rebe <rene@rocklinux.org>

+++ linux-2.4.21-rock/include/asm-i386/types.h.orig	2003-06-22 23:16:31.000000000 +0200
--- linux-2.4.21-rock/include/asm-i386/types.h	2003-06-22 23:16:22.000000000 +0200
@@ -17,7 +17,7 @@
 typedef __signed__ int __s32;
 typedef unsigned int __u32;
 
+#if defined(__GNUC__) && !defined(__STRICT_ANSI__)
-#if defined(__GNUC__)
 typedef __signed__ long long __s64;
 typedef unsigned long long __u64;
 #endif
